go.mod

```diff
@@ -35,21 +35,24 @@ require (
 	//
 	// when updating k8s.io/kubernetes, make sure to update the replace section too
 	//
-	k8s.io/api v0.26.3
-	k8s.io/apimachinery v0.26.3
+	k8s.io/api v0.27.0
```

Comment:
we have kubernetes 1.27.1 release , instead of updating to older release lets update to new release.

Reply:
Yes, sure.
Updated to 1.27.1